caffe2/operators/conv_op_eigen.cc

```diff
@@ -18,6 +18,12 @@
 #include "caffe2/core/operator.h"
 #include "caffe2/operators/conv_pool_op_base.h"
 
+#include "Eigen/Core"
+
+#if EIGEN_VERSION_AT_LEAST(3, 3, 0)
```

Comment:
I think you'd actually want

```
#if !EIGEN_VERSION_AT_LEAST(3, 3, 0)
#error "Caffe2 requires Eigen to be at least 3.3.0.";
#endif
```
Otherwise, yes please!

Reply:
lol yes, realized that error :P